core/src/com/fallenflame/game/LevelModel.java

```diff
@@ -172,9 +197,25 @@ public boolean inBounds(int x, int y) {
     public boolean isSafe(int x, int y) {
         //TODO: should return inBounds(x, y) and whether Tile.safe is true.
         //TODO: Tile.safe is false when it's an obstacle on it and true otherwise.
-        return (inBounds(x,y) && tileGrid[x][y].safe) || tileGrid[x][y].goal;
+        return (inBounds(x,y) && !(
+                        tileGrid[x][y].wall
+//                        tileGrid[x][y].enemy ||
+//                        tileGrid[x][y].player
+                )) || tileGrid[x][y].goal;
     } //TODO: temporary change
```

Comment:
What are the comments here for? IsSafe is used by AI, so it should be safe even if a player or the enemy calling it is there